src/components/ComplimentContainer/ComplimentContainer.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import "./ComplimentContainer.css"
+
+export interface ComplimentContainerProps {
+  compliment: string;
+}
+
+export const ComplimentContainer = ({
```

Comment:
Can you talk me through the difference between the ComplimentContainer and the AdviceContainer components?  Is there any chance we could combine the two to make it one dynamic contianer?